src/hourly-weather.ts

```diff
@@ -196,6 +196,11 @@ export class HourlyWeatherCard extends LitElement {
       }
     }
 
+    if (config.icon_fill && config.icon_fill !== 'single' && config.icon_fill !== 'full') {
+      if (!Number.isNaN(config.icon_fill) && (config.icon_fill < 1)) {
+        throw new Error(this.localize('errors.must_be_positive_int'));
```

Comment:
Let's not reuse this error string, as it won't be very helpful to users if they mistype the value. Can you make a new string in `src/localize/languages/en.json` under the `errors` section with something like this:

```
icon_fill must be either a positive integer or one of "single" or "full"
```

No need to translate to other languages. Once I merge this, I can trigger auto-translation on the new string before releasing.

Reply:
Got it. I also refactored the check to be more explicit about what is a valid value and what isn't.